extras/rya.pcj.fluo/pcj.fluo.api/src/main/java/org/apache/rya/indexing/pcj/fluo/api/GetQueryReport.java

```diff
@@ -132,14 +135,20 @@ private BigInteger countBindingSets(final SnapshotBase sx, final String nodeId,
         checkNotNull(nodeId);
         checkNotNull(bindingSetColumn);
 
+        NodeType type = NodeType.fromNodeId(nodeId).get();
+        Bytes prefixBytes = Bytes.of(type.getNodeTypePrefix());
+
         // Limit the scan to the binding set column and node id.
-        final RowScanner rows = sx.scanner().over(Span.prefix(nodeId)).fetch(bindingSetColumn).byRow().build();
```

Comment:
Sharding changes how binding set results can be accessed in the Fluo table.  There is no concern about breaking existing Fluo instances because the instances can't be updated.  Just redeployed with an updated jar.